themes/default/content/blog/migration-hub/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Pulumi adoption made easy with the new Migration Hub"
+allow_long_title: True
+authors: ["joe-duffy"]
+tags: ["opensource", "migration", "import"]
+meta_desc: "Introducing the Pulumi Migration Hub, your one stop shopping for migrating to Pulumi. It's never been so easy to adopt Pulumi."
+date: "2023-08-17"
+meta_image: "migration_hub.png"
+---
+
+Today we are launching Pulumi's new Migration Hub, a comprehensive guide to help you more seamlessly adopt Pulumi no
```

Comment:
@joeduffy Should we link to the Hub right here in the first paragraph? Otherwise it looks like the only link is in the final paragraph.

Reply:
Good call. Will do.